pkg/standalone/standalone.go

```diff
@@ -51,6 +51,13 @@ const (
 	redisDockerImageName       = "redislabs/rejson"
 	zipkinDockerImageName      = "openzipkin/zipkin"
 
+	githubContainerRegistryName = "ghcr"
+	ghcrURL                     = "ghcr.io/dapr"
```

Comment:
Variables like `ghcrURL` are more of URI and less of URL. How about using URI instead of URL